app/components/embedded_bindings/button_binding/button_binding_text.tsx

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Text, View, StyleSheet, StyleProp, TextStyle} from 'react-native';
+import Emoji from 'app/components/emoji';
+import {getEmoticonName} from 'app/utils/emoji_utils';
+
+// reEmoji matches an emoji (eg. :taco:) at the start of a string.
+const reEmoji = /^:([a-z0-9_\-+]+):\B/i;
```

Comment:
can we reuse the ones from `action_button_text.js` (or import them from some common file)?

Reply:
For the time being, as with App forms, I think we should keep things different to not break the previous support and not be chained by that.
That being said, right now I don't foresee much changes in the button text.